src/network/websockets/server/index.ts

```diff
@@ -44,6 +44,22 @@ export function runWebsocketServer(params: Params) {
 
     const handlers: { [action: string]: (msg: any) => void } = {
       ping: () => send({ action: "pong" }),
+      auth: msg => {
+        const { token } = msg;
+        onAuthenticate({ token })
+          .then(function(result) {
+            send({
+              action: "authResult",
```

Comment:
should these strings be dashed? That is an honest question cc @denis-sokolov

Reply:
Other protocol strings are camel case. 🤷‍♂

Reply:
Camel case seems consistent with the existing protocol spec (i.e. `pushResult`).

Reply:
cool thanks for pointing out we are already using a specific format.